src/corner_kick/README.md

```diff
@@ -1,6 +1,22 @@
-# Corner Kick
+<p align="center">
+  <img src="./src/assets/logo.svg" width="64px">
+  <br>
+  <h1 align="center">Corner Kick</h1>
+</p>
 
-Corner Kick is an Electron app that allows for the visualization of UBC Thunderbots' simulator.
+Corner Kick is a web app that allows for the visualization of UBC Thunderbots' AI.
+
+![application screenshot](screenshot.png)
+
+**Table of content**
```

Comment:
Could we add a small section explaining portals? Especially given that these are something new, custom, and specific to our team.

Reply:
```
#### React Portals

Access to those section in the React application is done via [React Portals](https://reactjs.org/docs/portals.html).
Portals are used to ensure the application layout is enforced throughout all the pages while offering an
easy way to access them.

In essence then, the sections of the application are boxes where the React application can insert React components into.
This allows us to insert and swap UI elements in the application without worrying about the global layout.
```

Reply:
This is what I have for that

Reply:
OH! I totally misunderstood when you explained this to me. I thought this was a totally new concept that you had come up with. My bad. Either way, this section is definitely good, so let's leave it!